flixel/group/FlxSpriteGroup.hx

```diff
@@ -280,6 +281,9 @@ class FlxTypedSpriteGroup<T:FlxSprite> extends FlxSprite
 		sprite.alpha *= alpha;
 		sprite.scrollFactor.copyFrom(scrollFactor);
 		sprite.cameras = _cameras; // _cameras instead of cameras because get_cameras() will not return null
+		
+		if (clipRect != null) clipRectTransform(sprite, clipRect);
+		else sprite.clipRect = null;
```

Comment:
I guess the alternative would be to merge the parent's clip and the child's clip together. I'd have to look at that in Flash to see how it works. Either way, the child's clip would change, though.